### PR TITLE
Update contributor guidelines and clarify PR review process

### PR DESCRIPTION
**Pull request by susodapop**
## What type of PR is this? (check all applicable)

- [x] Other

## Description

Update the PR template and CONTRIBUTING.md for 2022 with new SLA.

## Related Tickets & Documents

N/A

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -32,33 +20,70 @@ The following is a set of guidelines for contributing to Redash. These are guide
 - [Release Method](#release-method)
 - [Code of Conduct](#code-of-conduct)
 
+## Quick Links:
+
+- [Feature Requests](https://discuss.redash.io/c/feature-requests)
+- [Documentation](https://redash.io/help/)
+- [Blog](https://blog.redash.io/)
+- [Twitter](https://twitter.com/getredash)
```
Comment (arikfr) [inline CONTRIBUTING.md:28]: I would replace these two with the forum.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -32,33 +20,70 @@ The following is a set of guidelines for contributing to Redash. These are guide
...
+
+---
 ## How can I contribute?
 
-### Reporting Bugs
+## Reporting Bugs
```
Comment (arikfr) [inline CONTRIBUTING.md:33]: Why reduce the level? It's a sub headline after "How can I contribute?". No?

Comment (susodapop) [reply CONTRIBUTING.md]: I reverted this change.

File: CONTRIBUTING.md (hunk 1 above)
Comment (susodapop) [reply CONTRIBUTING.md]: Fixed [3edc967](https://github.com/getredash/redash/pull/5714/commits/3edc967ba74b3b826975e61433c2cd5311de90a9)

**Review by arikfr — APPROVED** [review]

**Pull request merged**